exp-player/addon/helpers/breaklines.js

```diff
@@ -0,0 +1,10 @@
+// h/t http://stackoverflow.com/questions/12331077/does-handlebars-js-replace-newline-characters-with-br
```

Comment:
Yes, I have confirmed that applying a CSS style to a span will suffice. There is no need to write handlebars code that rewrites or transforms the original text; let the browser handle presentation.

Here is the page if line 41 of audioplayer uses your helper:
<img width="1082" alt="screen shot 2016-03-04 at 11 27 34 pm" src="https://cloud.githubusercontent.com/assets/2957073/13545671/8181451e-e261-11e5-9a00-99ce132dc0a9.png">

Compared to the page if line 41 says `<span style="white-space:pre-line">{{prompt.text}}</span>`:
<img width="1173" alt="screen shot 2016-03-04 at 11 31 56 pm" src="https://cloud.githubusercontent.com/assets/2957073/13545693/2381d45a-e262-11e5-8f43-9645d77d3c65.png">